src/Trains.NET.Rendering/Game/Game.cs

```diff
@@ -148,21 +139,20 @@ public void DrawFrame()
 
                 using IImageCanvas imageCanvas = _imageFactory.CreateImageCanvas(_width, _height);
 
+                imageCanvas.Canvas.Save();
                 RenderFrame(imageCanvas.Canvas, pixelMapper);
-
-                _imageCache.Set(this, imageCanvas.Render());
+                imageCanvas.Canvas.Restore();
```

Comment:
I added a thing so this can be in a using block if you like
```
using (var _ = imageCanvas.Canvas.Scope())
{
    RenderFrame(...);
}
```

Reply:
Dropped this in slack as well, thoughts on also adding Measure?
![image](https://user-images.githubusercontent.com/20675893/94216174-18e73d00-ff22-11ea-8c2b-1bd51e85aca0.png)
Also, need to drop the `var` otherwise it sets the variable "_".

Reply:
Sounds good to me